kademlia/routing.py

```diff
@@ -67,6 +68,9 @@ def add_node(self, node):
             if node.id in self.replacement_nodes:
                 del self.replacement_nodes[node.id]
             self.replacement_nodes[node.id] = node
+            max_replacement_nodes = self.ksize * self.replacement_node_factor
```

Comment:
Thanks @JohnDoee!  Just one quick comment - what do you think about making `max_replacement_nodes` an instance variable instead of `self.replacement_node_factor`?  That way we could do this multiplication once in the constructor rather than every time we deal with the replacement list.

Reply:
I'll just merge and do in a subsequent commit.  Thanks!